intranet/templates/bus/bus_base.html

```diff
@@ -0,0 +1,34 @@
+{% load static %}
```

Comment:
I would either a) make this a full-blown base template and have the two pages `extends` it or 2) name this file `bus_head.html`. More descriptive.